### ISSUE TITLE
Error in man page gnt-cluster

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#170]
Originally reported of Google Code with ID 114.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".

gnt-cluster (ganeti) 2.1.6

Software version: 2.1.6
Internode protocol: 30
Configuration format: 2010000
OS api version: 15
Export interface: 0

<b>What distribution are you using?</b>

Ubuntu 10.04

<b>What steps will reproduce the problem?</b>
1. setup ganeti from source
2. read the man page gnt-cluster, INIT section

<b>What is the expected output? What do you see instead?</b>

Man page mentions: 
--master-netdev vg-name 

this should be
--master-netdev bridge-name


```
Originally added on 2010-08-12 13:26:03 +0000 UTC.

Attachments: 
[20100812152540.png](https://storage.googleapis.com/google-code-attachments/ganeti/issue-114/comment-0/20100812152540.png)

**Comment by Ganeti-Issues-Migrator** [comment]
```
Thanks a lot for the report. It should actually say: "interface-name" as the master netdev doesn't necessarily have to be a bridge. I sent a patch to update the manpage, and this will be fixed in our next stable release.
```
Originally added on 2010-08-12 15:53:14 +0000 UTC.

Added Labels: Type-Defect Priority-Low
Changed State: Accepted
Added to Milestone: Release2.1

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been fixed in branch-2.1 and will be included in the next stable release

Thanks,

Guido

```
Originally added on 2010-08-18 22:25:05 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:08 +0000 UTC.

Changed State: Released
